src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Effects/BitmapEffect.cs

```diff
@@ -85,7 +79,6 @@ protected BitmapEffect()
         [Obsolete(MS.Internal.Media.VisualTreeUtils.BitmapEffectObsoleteMessage)]
         unsafe static protected void SetValue(SafeHandle effect, string propertyName, object value)
```

Comment:
can this method be removed?

Reply:
Don't think so due to a similar reason as above